src/modules/ordersTable/pure/OrdersTableContainer/OrdersTable.tsx

```diff
@@ -215,7 +217,46 @@ export function OrdersTable({
   const checkboxRef = useRef<HTMLInputElement>(null)
 
   const step = currentPageNumber * ORDERS_TABLE_PAGE_SIZE
-  const ordersPage = orders.slice(step - ORDERS_TABLE_PAGE_SIZE, step).sort(ordersSorter)
+
+  const composableCowOrders = orders
+    .filter((order) => getIsComposableCowParentOrder(order))
+    .reduce<{ [key: string]: ParsedOrder }>((acc, val) => {
+      const parentId = val.composableCowInfo!.id!
+      acc[parentId] = val
+      return acc
+    }, {})
+
+  const partsOrders = orders
+    .filter((order) => getIsComposableCowChildOrder(order))
+    .reduce<{ [key: string]: ParsedOrder[] }>((acc, val) => {
```

Comment:
Nitpick: can you do filter inside the reduce?

Reply:
Nice suggestion! Thanks!